README.md

```diff
@@ -65,13 +65,13 @@ If you've never installed `backplane-tools` before, you will likely see a warnin
 > [!NOTE]
 > By default, backplane-tools and the applications it manages are not added to your `$PATH`. This avoids making assumptions on how users prefer to set their `$PATH` (ie- some users may prefer to `source` a different configuration file in their shell's .rc file), and, consequently, avoid unintentional collisions between the tools backplane-tools manages and those the user self-manages. This way, if you one day wish to take over the management of a certain utility, you can easily choose to invoke that instead by placing its location earlier on your `$PATH` than backplane-tools' entry.
```

Comment:
With this new approach, I wonder if that note is not confusing and should not be changed... WDYT ?

Reply:
What do you think makes it confusing?  I think this note makes sense. Per default, PATH is not changed, and the steps to change it are explained after.